### PR TITLE
feat: wire up vault staking asset card

### PR DESCRIPTION
**Pull request by technojak**
## Description

feat: wire up vault staking asset card

## Notice

Before submitting a pull request, please make sure you have answered the following:

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?
- [x] Do all new and existing tests pass? Does the linter pass?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

If applicable, please link to the github issue and put `closes #XXXX` in your comment to auto-close the issue that your PR fixes.

### CODE REVIEW

**Review by theoboldfrazier — APPROVED** [review]
LGTM

File: src/assets/translations/messages-en.json [hunk 1]
```diff
@@ -1,344 +1,350 @@
...
+    "approveAsset": "Approve interacting with %{symbol}",
+    "needPermission": "We need your permission to use your %{symbol}",
+    "whyNeedThis": "Why do I need to do this?",
+    "estimatedGasFee": "Estimated Gas Fee",
+    "approvingAsset": "Approving %{symbol}",
+    "viewTransaction": "View Transaction",
+    "tooltip": {
+      "rate": "This is the rate",
+      "minerFee": "This is the Miner Fee",
+      "shapeshiftFee": "This is the Shapeshift Fee"
```
Comment (0xdef1cafe) [inline src/assets/translations/messages-en.json:83]:
```suggestion
      "shapeshiftFee": "This is the ShapeShift fee"
```

File: src/assets/translations/messages-en.json [hunk 2]
```diff
@@ -1,344 +1,350 @@
...
-        "insufficientFundsForLimit": "Insufficient funds. The minimum trade amount for this pair is %{minLimit}",
-        "insufficientFundsForAmount": "Insufficient funds. Your %{symbol} balance is less than the selected amount",
-        "noLiquidityError": "Not enough liquidity available",
-        "overMaxSlippage": "This trade would result in over %{slippagePercentage}% slippage. Please trade a lower amount for a better price",
-        "balanceToLow": "Balance too low. The minimum trade amount for this pair is %{minLimit}",
-        "dexTradeFailed": "Trade Failed, Tokens Returned",
-        "quoteFailed": "An error occurred getting a quote for your trade."
-      }
+    "approve": {
+      "header": "Allow ShapeShift.io to Spend Your %{asset}",
```
Comment (0xdef1cafe) [inline src/assets/translations/messages-en.json]:
```suggestion
      "header": "Allow ShapeShift to Spend Your %{asset}",
```

File: src/assets/translations/messages-en.json (hunk 1 above)
Comment (technojak) [reply src/assets/translations/messages-en.json]: why would fee be capitalized? also Github is being jank and making this look like there is a massive diff even though only 4 phrases were added.

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**